superflore/utils.py

```diff
@@ -24,23 +24,23 @@
 from termcolor import colored
 
 
-def warn(string):
+def warn(string):  # pragma: no cover
     print(colored('>>>> {0}'.format(string), 'yellow'))
 
 
-def ok(string):
+def ok(string):  # pragma: no cover
     print(colored('>>>> {0}'.format(string), 'green'))
 
 
-def err(string):
+def err(string):  # pragma: no cover
     print(colored('!!!! {0}'.format(string), 'red'))
 
 
-def info(string):
+def info(string):  # pragma: no cover
     print(colored('>>>> {0}'.format(string), 'cyan'))
 
 
-def make_dir(dirname):
+def make_dir(dirname):  # pragma: no cover
```

Comment:
This could be trivially tested in a tempdir. It would be good to add tests for the different edge cases such as the directory existing, not existing, is file, and not writable.

Reply:
That's a good point. Will do.